### ISSUE TITLE
SomeCommentsAboutTheCode0526

### ISSUE THREAD

**Issue by liuzrcc** [patricksttan/Iamthinking#5]
There  are 3 models in the code.

The first one whose name is create_model_v1.

The second one named create_model_liu1.

The third one is vgg_std16_model.

The first one is the one we took from the forums. The second one I got it from keras website, it already run but the result is not good with a log loss around 3. The third one is the vgg16 model. Our aim now is to let this model run for our data.

To run the last model, we need the weights [vgg16_weights file](https://drive.google.com/file/d/0Bz7KyqmuGsilT0J5dmRCM0ROVHc/view?pref=2&pli=1)

For the keras information, it's here.(VGG-like convnet already run successfully)[keras vgg information](http://keras.io/getting-started/sequential-model-guide/)
